addons/sale_project/models/sale_order_line.py

```diff
@@ -144,7 +144,16 @@ def _timesheet_create_project(self):
 
         # Avoid new tasks to go to 'Undefined Stage'
         if not project.type_ids:
-            project.type_ids = self.env['project.task.type'].create({'name': _('New')})
+            project.type_ids = self.env['project.task.type'].create([{
+                'name': name,
+                'fold': fold,
+                'sequence': sequence,
+            } for name, fold, sequence in [
+                (_('To Do'), False, 5),
+                (_('In Progress'), False, 10),
+                (_('Done'), True, 15),
+                (_('Canceled'), True, 20),
```

Comment:
I will check with Alexandra if it is now really necessary to create those stages. 🙂

Reply:
hey @xavierbol thank you for the review, have you discussed the centered div (previous comment) and this points with AJU ?

Reply:
Hello @RaoufGhrissi , yes you can see the discussion in the chatter of the task.

We will not do the centered div but we will just decrease the padding top (if needed)

For the stage, you can let as you did. 🙂